pkg/api/handlers/handler.go

```diff
@@ -1,6 +1,14 @@
 package handlers
 
-const (
-	DefaultApiVersion = "1.40" // See https://docs.docker.com/engine/api/v1.40/
-	MinimalApiVersion = "1.24"
+import (
+	"github.com/blang/semver"
+)
+
+var (
+	// See https://docs.docker.com/engine/api/v1.40/
+	// libpod compat handlers are expected to honor docker API versions
+	MaxApiVersion     = semver.MustParse("1.40.0") // Maximum supported Libpod API version must be >= docker API
```

Comment:
Should we split this for Compat and Libpod? I don't know if I want us to be beholden to Docker versioning for Libpod

Reply:
If we don't at least draft with them, we're creating a blocking change moving between compat and libpod.

Reply:
I think we'll already have problems with people checking Docker version (Podman v2.0.0 vs Docker 19.05), so we can't guarantee full compatibility. Though this is probably a conversation for watercooler tomorrow.

Reply:
I won't be able to join the water cooler, so I am sharing my thoughts here.

I think it's important to be able (client & server) to communicate which versions of Docker are supported and which version of Podman. If they overlap, clients may know if the server actually supports what they want and the server may not be able to tell the client if they are supported or not. Mixing the two or letting them overlap would erase (to a certain degree) the expressiveness of semver.

If a user/costumer wants to figure out which versions (of Docker) a given Podman supports, they should see that with a simple `podman info` which could list a version range for Docker and another one for the Podman/libpod endpoints. If they overlapped, the data would not answer their question. This would make the lives of support much easier, especially for customer cases.